Cant install package

Hey, I have started a new chakra ui project (it is 1.8.8 at the moment of writing) and npm gives me the following error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: client@0.1.0
npm ERR! Found: framer-motion@6.3.3
npm ERR! node_modules/framer-motion
npm ERR!   framer-motion@"^6.3.3" from the root project
npm ERR!   peer framer-motion@"3.x || 4.x || 5.x || 6.x" from @chakra-ui/react@1.8.8
npm ERR!   node_modules/@chakra-ui/react
npm ERR!     @chakra-ui/react@"^1.8.8" from the root project
npm ERR!     peer @chakra-ui/react@">=1.6.7" from chakra-ui-steps@1.7.2
npm ERR!     node_modules/chakra-ui-steps
npm ERR!       chakra-ui-steps@"*" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer framer-motion@"^5" from chakra-ui-steps@1.7.2
npm ERR! node_modules/chakra-ui-steps
npm ERR!   chakra-ui-steps@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/chakra-ui-steps/package.json
+++ b/chakra-ui-steps/package.json
@@ -38,7 +38,7 @@
     "@chakra-ui/react": ">=1.6.7",
     "@emotion/react": "^11",
     "@emotion/styled": "^11",
-    "framer-motion": "^5",
+    "framer-motion": "^6",
     "react": ">=16"
   },
   "husky": {

